edx_lint/pylint/unicode_check.py

```diff
@@ -84,4 +97,12 @@ def process_tokens(self, tokens):
                     # Valid JSON isn't human-readable, leave it alone.
                     continue    # pragma: no cover   grr.. https://github.com/nedbat/coveragepy/issues/198
 
+                if re.match(r"^(def|class) [a-zA-Z0-9_]*\(.*", previous_line):
+                    # Previous line probably contains a function or class definition
+                    # so this string is probably a docstring
+                    continue
+                if index == 0:
+                    # If this string is the first line of the file, it is likely a docstring
```

Comment:
"If this string is the first token of the file, it is definitely a docstring"

But the docstring can appear after a comment on the first line, so it isn't necessarily the first token.

Reply:
good point, added code to check for comments before the file docstring